pubspec.yaml

```diff
@@ -77,23 +81,31 @@ dependencies:
   emoji_picker_flutter: ^1.5.1
   emoji_proposal: ^0.0.1
   emojis: ^0.9.9
-  fcm_shared_isolate: ^0.1.0
+  # TODO: remove this after this PR is merged https://gitlab.com/famedly/company/frontend/libraries/native_imaging/-/merge_requests/22
+  fcm_shared_isolate:
+    git:
+      url: https://github.com/famedly/fcm_shared_isolate.git
+      ref: main
   file_picker: ^8.0.5
   flutter:
     sdk: flutter
-  flutter_app_badger: ^1.5.0
+  # TODO: remove this after this PR is merged https://github.com/g123k/flutter_app_badger/pull/92
+  flutter_app_badger:
+    git:
+      url: https://github.com/bitsydarel/flutter_app_badger.git
+      ref: master
   flutter_app_lock: ^3.0.0
   flutter_blurhash: ^0.8.2
   flutter_cache_manager: ^3.3.0
   flutter_foreground_task: ^3.10.0
-  flutter_local_notifications: ^17.1.2
+  flutter_local_notifications: 17.2.2
```

Comment:
- [ ] demo related to notification in Android

Reply:
@nqhhdev hi, can you add me the demo of noti in android, please

Reply:
thanks you

Reply:
Done !

Reply:
how can i find the demo?